components/expenses/ConfirmProcessExpenseModal.tsx

```diff
@@ -236,9 +242,19 @@ export default function ConfirmProcessExpenseModal({ type, onClose, expense }: C
     <StyledModal role="alertdialog" width="432px" onClose={onClose} trapFocus>
       <ModalHeader>{intl.formatMessage(MessagesPerType[type].title)}</ModalHeader>
       <ModalBody pt={2}>
-        <P mb={3} color="black.700" lineHeight="20px">
-          {intl.formatMessage(MessagesPerType[type].description)}
-        </P>
+        <Flex>
+          <AlertTriangleIcon size={30} color="orange" />
+          <P mb={3} color="black.700" lineHeight="20px" ml={10}>
+            {intl.formatMessage(MessagesPerType[type].description)}
+          </P>
+        </Flex>
+
+        {MessagesPerType[type].label && (
+          <P mb={2} color="black.700" fontWeight="500">
+            {intl.formatMessage(MessagesPerType[type].label)}
```

Comment:
Let's also clarify that it's optional:

```suggestion
            {intl.formatMessage({ id: 'OptionalFieldLabel', defaultMessage: "{field} (optional)" }, {
              field: intl.formatMessage(MessagesPerType[type].label)
            )}
```